src/playwrightTest.ts

```diff
@@ -63,11 +63,21 @@ export class PlaywrightTest {
     this._envProvider = envProvider;
   }
 
-  async getPlaywrightInfo(workspaceFolder: string, configFilePath: string): Promise<{ version: number, cli: string }> {
+  private async getPlaywrightInfo(workspaceFolder: string, configFilePath: string): Promise<{ version: number, cli: string }> {
+    try {
+      return await this._getPlaywrightInfoImpl(workspaceFolder, configFilePath, '@playwright/test');
+    } catch (error) {
+      return await this._getPlaywrightInfoImpl(workspaceFolder, configFilePath, '@playwright/experimental-ct-core');
+    }
+  }
+
+  private async _getPlaywrightInfoImpl(workspaceFolder: string, configFilePath: string, cliPackage: string): Promise<{ version: number, cli: string }> {
     const pwtInfo = await this._runNode([
       '-e',
-      'try { const pwtIndex = require.resolve("@playwright/test"); const version = require("@playwright/test/package.json").version; console.log(JSON.stringify({ pwtIndex, version})); } catch { console.log("undefined"); }',
+      `try { const pwtIndex = require.resolve("${cliPackage}"); const version = require("${cliPackage}/package.json").version; console.log(JSON.stringify({ pwtIndex, version})); } catch { console.log("null"); }`,
```

Comment:
Note: The reason for changing `undefined -> null` was that `JSON.parse('undefined')` does not work while it does with `null`. Added a null check afterwards to be on the safe side.